web/turkey/survey/static/survey/js/auditors/keypresses_total.js

```diff
@@ -0,0 +1,25 @@
+var AuditorKeypressesTotal = {
+    keypresses_total: 0,
+    log_keypress_event: function (e) {
+        // e.stopPropagation();
+        this.keypresses_total += 1;
+    },
+    submit_callable: function () {
+        return {
+            'count': this.keypresses_total
+        };
+    }
+};
+
+var auditor_keypresses_total = Object.create(AuditorKeypressesTotal);
+
+$(document).ready(function() {
+    $(document).keydown(auditor_keypresses_total
```

Comment:
Does keydown get called multiple times if someone is say, holding the down key? If so, do we want to count that as multiple presses?